### DISCUSSION TITLE
How to connect redis cloud

### DISCUSSION THREAD

**Discussion by quyenvnk63** [vercel/next.js#52905] [Help]
### Summary

Hi everybody. I have a problem as follows. i use bull redis to create a scheduler function. I have configured the code as follows 
[ const redis = require('redis');
const { promisify } = require('util');

require('dotenv').config();
// Create a Redis client
const redisClient = redis.createClient({
  host: process.env.REDIS_HOST || '127.0.0.1',
  port: process.env.REDIS_PORT || 6379,
  password: process.env.REDIS_PASSWORD ,
});]. After I deploy the code to git and configure the environment variables obtained from redis cloud including: REDIS_HOST,REDIS_PORT,REDIS_PASSWORD on vercel. but it seems the back end is still trying to connect to the local redis server. The error is reported by vercel as follows 

### Additional information

```js
[Error connecting to Redis: Error: connect ECONNREFUSED 127.0.0.1:6379
     at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1494:16) {
   errno: -111,
   code: 'ECONREFUSED',
   syscall: 'connect',
   address: '127.0.0.1',
   port: 6379
}]. I look forward to help. thank you
```

**Comment by MiguelTrinca** [comment]
I had a similar problem, but I just used the redis cloud variables and it worked. Maybe remove the "127.0.0.1" and "6379". Also make sure this is the only place you are calling the redis client
